File: hw_OOP_06/View/ViewConsole.java
package View;

import java.util.Scanner;

public class ViewConsole implements ViewRealization {

    @Override
    public String getInfixExpression() {
        Scanner scanner = new Scanner(System.in);
        System.out.printf("Введите выражение без пробелов: ");
        String expression = scanner.next();
        scanner.close();
        return expression;
    }

    @Override
    public void resultV(double result) {
        System.out.printf("result = %f\n", result);
    }

}

File: hw_OOP_07/ViewConsole.cs
class ViewConsole : ViewAbstrct
{

    override
    public string getInfixExpression() 
    {
        Console.Write("Введите выражение без пробелов: ");
        string? expression = Console.ReadLine();
        return expression;
    }

    override
    public void resultV(double result) 
    {
        Console.WriteLine($"result = {result}");
    }

}
